### ISSUE TITLE
The admin interface password field must request the password twice and compare that are the same

### ISSUE THREAD

**Issue by fpietrosanti** [globaleaks/GlobaLeaks#1598]
The admin interface password field must request the password twice and compare that are the same.

Actually the password is asked in the User profile under User Management only once, so the admin may mistake typing it.

**Comment by evilaliv3** [comment]
For me is not a bug, as i designed it that way intentionally; 

Generally administrars does not have the confirmatiion for the password when i apply to other users; If the admin type it wrong anyhow he will be able to set it a second time.

But we can eventually change this if this gives you headache!

The problem is when you are changing your password and you lockout yourself.

**Comment by fpietrosanti** [comment]
It's always a requirements for admin interfaces to prevent errors ask the password twice, look at Google Apps admin interface to create user and reset password of users:
<img width="622" alt="screen shot 2016-03-22 at 3 13 08 pm" src="https://cloud.githubusercontent.com/assets/770363/13954365/b1eabdb0-f040-11e5-9006-c7cbfb6b4794.png">
<img width="616" alt="screen shot 2016-03-22 at 3 12 23 pm" src="https://cloud.githubusercontent.com/assets/770363/13954366/b1eb2be2-f040-11e5-996d-5cfc34f14a20.png">

**Comment by evilaliv3** [comment]
It depends, not all the interfaces do but ok we can have this change.

you want it also for the SMTP password? (in this case gmail does not ask it :P)

**Comment by fpietrosanti** [comment]
No, for SMTP password, the error prevention approach is to implement #1565

**Comment by evilaliv3** [comment]
@NSkelsey would you like to keep this ticket in your bucket? it's an other small fix that reusing existing code would help you understand the technologies used in the project and their integration.
